apcd-cms/src/apps/admin_regis_table/templates/list_registrations.html

```diff
@@ -49,5 +60,22 @@ <h1>List Registrations</h1>
       </tbody>
   </table>
 </div>
+<script>
+  function filterTableByStatus() {
+    var dropdown, filter, table, rows, status, i;
+    dropdown = document.getElementById("statusFilter");
+    filter =  dropdown.value;
+    table = document.getElementById("registrationTable");
+    rows = table.getElementsByTagName("tr");
+    for (i = 1; i < rows.length; i++) {
```

Comment:
Nowadays, one can use [Spread syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) (and maybe [`Array.from`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from)) to make an array from a DOM node list.

```
[ ...rows ].forEach( row => {
    ...
});
```